### PR TITLE
Amam/hotfix_update

### PR DESCRIPTION
**Pull request by mustofa-binary**
# Description

- fix url for pdfs in payment methods
- fix url parameters to cta button

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/localization/localized-link.js [hunk 1]
```diff
@@ -65,7 +65,14 @@ export const LocalizedLink = ({ to, ...props }) => {
 
     if (props.external || props.external === 'true') {
         return (
-            <a target={target} rel={rel} className={className} style={style} href={to}>
+            <a
+                target={target}
+                rel={rel}
+                className={className}
+                data-amp-replace="QUERY_PARAM"
```
Comment (oskar-binary) [inline src/components/localization/localized-link.js:72]: what is `data-amp-replace`?

File: src/pages/amp/homepage/_hero.js [hunk 2]
```diff
@@ -98,8 +98,11 @@ const Hero = () => {
                 </Box>
                 <amp-img src={HeroPlatform} width="540" height="345" />
             </FullWidth>
-
-            <BigButton secondary to="/signup/" target="_blank" rel="noopener noreferrer">
+            <BigButton
+                secondary
+                to="/signup/?t=QUERY_PARAM(t)&utm_source=QUERY_PARAM(utm_source)&utm_medium=QUERY_PARAM(utm_medium)&utm_campaign=QUERY_PARAM(utm_campaign)"
```
Comment (oskar-binary) [inline src/pages/amp/homepage/_hero.js:103]: for `to` we could add it to a function and use it here and in `MediumButton`?

File: src/components/localization/localized-link.js (hunk 1 above)
Comment (mustofa-binary) [reply src/components/localization/localized-link.js:72]: this is a built in amp function, that will redirect query parameter to non amp page, amp page cant access window, localstorage, cookies, etc. so this is the only option to redirect the query parameter

Comment (oskar-binary) [reply src/components/localization/localized-link.js:72]: oh I see, thanks for clarifying @mustofa-binary

**Pull request merged**